content/en/blog/OWNERS

```diff
@@ -1,7 +1,8 @@
 # Owned by Kubernetes blog reviewers.
 options:
   no_parent_owners: true
-# reviewers: // List any reviewers here
+# reviewers:
```

Comment:
I think this line (and the same line in the next file) will be ignored because of the leading comment characters?

Reply:
@mistyhacks Fixed.